app/Http/Controllers/Store/BundleController.php

```diff
@@ -51,7 +52,65 @@ public function index()
     }
 
     /**
-     * Create OR Update a registrations current active bundle
+     * Does a single voucher.
+     *
+     * @param StoreAppendBundleRequest $request
+     * @param Registration $registration
+     * @return \Illuminate\Http\RedirectResponse
+     */
+
+    public function addVouchersToCurrentBundle(StoreAppendBundleRequest $request, Registration $registration)
```

Comment:
Consider sharing some functionality with `/app/Http/Controllers/Service/Admin/VouchersController.php`

The diff is that submit comes with Sponsor id and numbers - but might be able to use some rules and `range()`